editor/editor.cpp

```diff
@@ -39,22 +34,39 @@ void Editor::initialize(HWND hWnd)
 		(float)general["colors"]["inactive"]["b"],
 		(float)general["colors"]["inactive"]["a"],
 	};
-
-	LuaVariable viewport = LuaInterpreter::GetSingleton()->getGlobal("viewport");
-	m_ViewportSettings.m_AspectRatio = (float)LuaInterpreter::GetSingleton()->getGlobal("viewport")["aspectRatio"];
-	m_ViewportSettings.m_ImageTint = {
-		(float)viewport["imageTint"]["r"],
-		(float)viewport["imageTint"]["g"],
-		(float)viewport["imageTint"]["b"],
-		(float)viewport["imageTint"]["a"],
+	m_Colors.m_Success = {
```

Comment:
I see a lot of redundant code here. Can't we use arrays and for loops to assign colors here?